src/adapters/Base/index.test.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable camelcase */
+import requestManager from 'lib/request/v1/requestManager';
```

Comment:
I am curious why you want `version`ing here?

Reply:
@tyrro it's the suggestion from @nvminhtue [here](https://github.com/manh-t/react-survey/pull/46#discussion_r1252527159) 🙏